daemon/policy_test.go

```diff
@@ -444,12 +444,15 @@ func (ds *DaemonSuite) TestRemovePolicy(c *C) {
 		os.RemoveAll("1")
 		os.RemoveAll("1_backup")
 	}()
+	buildSuccess := <-e.Regenerate(ds.d, "test")
+	c.Assert(buildSuccess, Equals, true)
+
```

Comment:
IIRC, that's because we now have to go through 2 generations to have the policy generated for the endpoint's final identity. The 1st generation is for the `reserved:init` identity, the 2nd is for the final identity.

Reply:
I removed this. It is not needed anymore.